packages/auth/package.json

```
@@ -88,7 +88,7 @@
 		"@aws-amplify/core": "^6.0.0"
 	},
 	"devDependencies": {
-		"@aws-amplify/core": "6.0.0",
+		"@aws-amplify/core": "^6.0.0",
```

```suggestion
		"@aws-amplify/core": "6.0.0",
```

The dev dep shouldn't be pinned, it's for our internal dev and lerna will handle it